addons/voxel-core/classes/readers/vox.gd

```diff
@@ -365,13 +365,23 @@ static func _unpack_transfom(rotation, translation) -> Transform:
 			row2 = -row2
 		
 		# correct rotation for godot
+#		var eulerRot = Basis(row0, row1, row2).get_euler()
+#		var correctedBasis = Basis(
+#			Vector3(
+#				eulerRot.x,
+#				-eulerRot.z,
+#				-eulerRot.y
+#			)
+#		)
+
 		var eulerRot = Basis(row0, row1, row2).get_euler()
 		var correctedBasis = Basis(
 			Vector3(
 				eulerRot.x,
 				-eulerRot.z,
 				-eulerRot.y
-			)
+			),
+			0
```

Comment:
This should probably use the `from_euler()` method instead of using an axis:
```gdscript
		var correctedBasis = Basis().from_euler(
			Vector3(
				eulerRot.x,
				-eulerRot.z,
				-eulerRot.y
			)
		)
```